python/pyspark/sql/connect/window.py

```diff
@@ -201,7 +201,7 @@ def __repr__(self) -> str:
         return "WindowSpec(" + ", ".join(strs) + ")"
 
 
-WindowSpec.__doc__ = PySparkWindow.__doc__
+# WindowSpec.__doc__ = PySparkWindow.__doc__
```

Comment:
Disabling this because it runs the windowspec doctests

Reply:
Why don't you run them here?